build.gradle

```diff
@@ -24,4 +25,10 @@ dependencies {
 
 test {
     useJUnitPlatform()
-}
\ No newline at end of file
+}
+
+spotless {
+    java {
+        eclipse().configFile('meta/formatting/google-style-eclipse.xml')
```

Comment:
What do you think about adding `removeUnusedImports()`?

Reply:
Sounds fine :)